package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's remove this file; this is the result of executing `yarn` commands from repo's root directory instead of `ui` directory.